clients/rospy/src/rospy/rosconsole.py

```diff
@@ -135,13 +143,102 @@ def _rosconsole_cmd_get(argv):
     print(logger_level._current_levels[args[1]])
 
 
+class RosConsoleEcho(object):
+    # See ANSI/VT100 terminal color codes here:
+    # https://misc.flogisoft.com/bash/tip_colors_and_formatting
+    LEVEL_COLOR = {
+        'DEBUG': 92,  # Light green
+        'INFO' : 97,  # White
+        'WARN' : 93,  # Light yellow
+        'ERROR': 91,  # Light red
+        'FATAL': 95,  # Light magenta
+    }
+
+    def __init__(self, options):
+        self._filter = re.compile(options.filter)
+        self._level = getattr(Log, options.level.upper())
+        self._nocolor = options.nocolor
+        self._verbose = options.verbose
+
+        self._level_string_map = {getattr(Log, level): self._stringify(level) for level in self.LEVEL_COLOR.keys()}
+
+        callback = self._once_callback if options.once else self._callback
+        rospy.Subscriber(options.topic, Log, callback)
+
+    def _stringify(self, level):
+        string = level.ljust(5)
```

Comment:
@dirk-thomas I feel I should change this to `string = level.ljust(max([len(level) for level in LEVEL_COLOR.keys()]))`

Sounds good?

Reply:
Done.

Also rebased onto `lunar-devel`, although the tests were already passing before that.